build/bootstrap/bootstrap-unison.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+#
+# On a Debian/Ubuntu system, bootstraps the Unison file-syncer.
+
+set -euxo pipefail
+
+sudo apt-get install -y --no-install-recommends ocaml-nox
```

Comment:
`sudo apt-get install unison` worked for me. any reason for this elaborate bootstrap?

Reply:
Nope, I'm on ubuntu locally, so I used `apt-get install unison` and didn't run `bootstrap-unison.sh` locally. So I wanted to know why we couldn't do that remotely.